### PR TITLE
feedback fix

### PR DESCRIPTION
**Pull request by Vips19**

### CODE REVIEW

**Comment by Vips19** [comment]
#3773

File: src/projects/detail/containers/AssetsInfoContainer.jsx [hunk 1]
```diff
@@ -836,7 +836,7 @@ class AssetsInfoContainer extends React.Component {
             </div>
             {(showAddNewButton) && (
               <div styleName="assets-header-button">
-                <button type="button" onClick={newButtonClick} styleName="add-new-button">Add new...</button>
+                <button type="button" onClick={newButtonClick} styleName="add-new-button" className="tc-btn tc-btn-primary tc-btn-sm">Add new...</button>
```
Comment (maxceem) [inline src/projects/detail/containers/AssetsInfoContainer.jsx:839]: As now we use standard styles for buttons, can we remove `styleName="add-new-button"` and corresponding style from SCSS file, as we don't use it?

Comment (Vips19) [reply src/projects/detail/containers/AssetsInfoContainer.jsx:839]: @maxceem Yes we can remove the above class, I have also updated the same styles from _button.scss as mentioned by you. Thanks for your feedback

**Comment by Vips19** [comment]
@maxceem Thanks

**Pull request merged**